REST api testing/api_client.py

```diff
@@ -0,0 +1,10 @@
+import requests
+
+
+class APIClient:
+    def __init__(self, base_url):
+        self.base_url = base_url
+
+    def get_hotels(self):
+        response = requests.get(f"{self.base_url}/hotels")
+        return response.json()
```

Comment:
![image](https://github.com/nikita12012005/Pytest/assets/25909810/992d43f3-8178-4229-ba46-8dbca090d65e)
So many endpoint present in this API why do you use only 1? please provide 5+ endpoints